updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,145 @@
+You can update, or upgrade a restricted network {product-title} cluster using the `oc` command-line interface (CLI).
 
 .Prerequisites
 
+* Have access to the internet to obtain the necessary container images.
+* Have write access to a container registry in the restricted-network environment to push and pull images. The container registry should be compatible with Docker registry API v2.
+* You must have the `oc` command-line interface (CLI) tool installed.
 * Have access to the cluster as a user with `admin` privileges.
 See xref:../authentication/using-rbac.adoc[Using RBAC to define and apply permissions].
 * Have a recent xref:../backup_and_restore/backing-up-etcd.adoc#backup-etcd[etcd backup] in case your upgrade fails and you must xref:../backup_and_restore/disaster_recovery/scenario-2-restoring-cluster-state.adoc#dr-restoring-cluster-state[restore your cluster to a previous state].
 
-//mirror module
+[id="update-restricted-network"]
+== Updating a restricted network {product-title} cluster
 
-include::modules/update-service-overview.adoc[leveloffset=+1]
+A restricted network environment is the one in which your cluster nodes cannot access the internet. For this reason you must mirror the images to a filesystem disconnected from that environment and then bring that host or removable media across that gap.
 
-include::modules/understanding-upgrade-channels.adoc[leveloffset=+1]
+If multiple clusters are present within the restricted network, the recommended way is to mirror the required release images to a single container image registry and use it for updating all the clusters.
 
-include::modules/update-upgrading-web.adoc[leveloffset=+1]
+== Preparing your mirror host
+
+Before you perform the mirror procedure, you must prepare the host to retrieve content
+and push it to the remote location.
+
+include::modules/cli-installing-cli.adoc[leveloffset=+2]
+
+include::modules/installation-adding-mirror-registry-pull-secret.adoc[leveloffset=+1]
+
+[IMPORTANT]
+===
+This step requires that you have write access to a container image registry on the
+mirror registry and adding those credentials to a registry pull secret. This secret
+should NOT be used to install a cluster as it would give machines within the cluster
+write access to your mirror registry.
+===
+
+=== Mirror the images to local container registry
+
+Before you update a cluster on infrastructure that you provision in a restricted network, you must mirror the required container images into that environment. You may also use this procedure in unrestricted networks to ensure your clusters only use container images that have satisfied your organizational controls on external content.
+
+.Procedure
+
+. Find out if there is link:https://access.redhat.com/solutions/4583231[an update path] that exists between your current version and the desired version for the cluster to move.
+
+. Set the required environment variables:
++
+----
+$ OCP_RELEASE=<release_version> <1>
```

Comment:
These are coming out a `i`, `ii`, etc. Believe you need `# <1>`

Reply:
I tried to fix it but still getting issues. I will let doc team fix it :)